wazo_test_helpers/asset_launching_test_case.py

```diff
@@ -8,13 +9,26 @@
 import subprocess
 import tempfile
 import unittest
+from collections.abc import Callable, Generator
+from pathlib import Path
+
+from typing import TypeVar, Generic, Any, cast, NoReturn, TYPE_CHECKING, TextIO
 
 import docker as docker_client
+import pytest
+
+if TYPE_CHECKING:
+    from typing import ParamSpec
+    from tempfile import _TemporaryFileWrapper
+
+    P = ParamSpec('P')
+    C = TypeVar("C", bound=type[Any])
+    R = TypeVar('R')
```

Comment:
Would it be possible to make P,C and R more explicit? Not sure what they should mean, but something like `ParamType`, `CallableType`, `ReturnType`?

Reply:
We definitely can. I was trying to follow the examples, and despite pep8 all the examples, everywhere use one letter uppercase variables for these specific cases, `P` for `ParamSpec`, `R` for return value and `T` for standard `TypeVar`, the `C` was me, as it is a type and not the instance, so I used `C` for class. At least the latter I can rename `ClassType` or something.